descriptions.md

```diff
@@ -32,6 +31,15 @@ A client that offers a simple stateless API to interact directly with Ably's RES
 ||| `PaginatedResult` || A [`PaginatedResult`]{@link PaginatedResult} object containing an array of [`Stats`]{@link Stats} objects. |
 | time() => io Time ||| RSC16 | Retrieves the time from the Ably service as milliseconds since the Unix epoch. Clients that do not have access to a sufficiently well maintained time source and wish to issue Ably [`TokenRequest`s]{@link TokenRequest} with a more accurate timestamp should use the [`queryTime`]{@link ClientOptions#queryTime} property instead of this method. |
 ||| `Time` || The time as milliseconds since the Unix epoch. |
+| batchPublish(BatchPublishSpec) => io BatchResult<BatchPublishSuccessResult \| BatchPublishFailureResult> ||| RSC22 | Publishes a [`BatchPublishSpec`]{@link BatchPublishSpec} object to one or more channels, up to a maximum of 100 channels. |
+|| `BatchPublishSpec` ||| A [`BatchPublishSpec`]{@link BatchPublishSpec} object. |
+||| `BatchResult<BatchPublishSuccessResult \| BatchPublishSuccessResult>` || A [`BatchResult`]{@link BatchResult} object containing information about the result of the batch publish for each requested channel. |
+| batchPublish([BatchPublishSpec]) => io [BatchResult<BatchPublishSuccessResult \| BatchPublishFailureResult>] ||| RSC22 | Publishes a [`BatchPublishSpec`]{@link BatchPublishSpec} object to one or more channels, up to a maximum of 100 channels. |
+|| [`BatchPublishSpec`] ||| An array of [`BatchPublishSpec`]{@link BatchPublishSpec} objects. |
+||| [`BatchResult<BatchPublishSuccessResult \| BatchPublishSuccessResult>`] || An array of [`BatchResult`]{@link BatchResult} objects containing information about the result of the batch publish for each requested channel for each provided [`BatchPublishSpec`]{@link BatchPublishSpec}. The array returned is in the same order as the provided [`BatchPublishSpec`]{@link BatchPublishSpec} array. |
+| batchPresence([String]) => io BatchResult<BatchPresenceSuccessResult \| BatchPresenceFailureResult> ||| RSC23 | Retrieves the presence state for one or more channels, up to a maximum of 100 channels. Presence state includes the `clientId` of members and their current [`PresenceAction`]{@link PresenceAction}. |
+|| [String] ||| An array of one or more channel names, up to a maximum of 100 channels. |
+||| `BatchResult<BatchPresenceSuccessResult \| BatchPresenceFailureResult>` || A [`BatchResult`]{@link BatchResult} object containing information about the result of the batch presence request for each requested channel. |
```

Comment:
I think so, yes, although I'm not sure if this is a documented behaviour or not so it's probably best to recommend correlating requests to responses by matching on channel name. This is more important for token revocation and batch publishing since a `BatchPublishSpec[]` or `TokenRevocationTarget[]` could result in multiple success/failures for a single channel so matching on channel name wouldn't suffice.

Reply:
Ah ok, that makes sense!